src/thunks/handleProject.js

```diff
@@ -1,22 +1,23 @@
-import { isLoading, hasErrored, addProjectSuccess } from '../actions'
+import { isLoading, hasErrored } from '../actions'
 
-export const addProject = (url, project_name) => {
+export const handleProject = (url, actionToDispatch, method, project) => {
```

Comment:
Great work making this thunk more dynamic - makes a ton of sense to have this handle all of our async flows re: projects as opposed to a separate thunk for each method!